iceoryx_hoofs/test/moduletests/test_logstream.cpp

```diff
@@ -195,67 +164,105 @@ TEST_F(IoxLogStream_test, StreamOperatorLogRawBuffer)
     }
 }
 
+#endif
+
 template <class T>
-class IoxLogStreamHexBin_test : public IoxLogStream_test
+class IoxLogStreamHexOctBinIntegral_test : public IoxLogStream_test
 {
   public:
     T LogValueLow = std::numeric_limits<T>::lowest();
     T LogValueMin = std::numeric_limits<T>::min();
     T LogValueMax = std::numeric_limits<T>::max();
 };
 
-using LogHexBinTypes = Types<uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t>;
+using LogHexOctBinIntegralTypes = Types<uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t>;
 
 #ifdef __clang__
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
 #endif
-TYPED_TEST_SUITE(IoxLogStreamHexBin_test, LogHexBinTypes);
+TYPED_TEST_SUITE(IoxLogStreamHexOctBinIntegral_test, LogHexOctBinIntegralTypes);
```

Comment:
You can remove the suppressed warning `-Wgnu-zero-variadic-macro-arguments` when you add an additional comma here.
```cxx
TYPED_TEST_SUITE(IoxLogStreamHexOctBinIntegral_test, LogHexOctBinIntegralTypes, );
```

Reply:
oh, nice. We have 44 occurrences of `Wgnu-zero-variadic-macro-arguments`. Would it be okay for you when I create a separate issue and fix all of this at once?

Reply:
I created an issue: https://github.com/eclipse-iceoryx/iceoryx/issues/1653